articles/connectors/connectors-native-http.md

```diff
@@ -302,6 +302,17 @@ HTTP requests have a [timeout limit](../logic-apps/logic-apps-limits-and-config.
 
 <a name="disable-location-header-check"></a>
 
+### Retry-After  
+The http action respects the retry after with 429 status code if received one .
+the value will be considered as Seconds
+
```

Comment:
This example doesn't seem to follow the documented format. Will get back to you on this.

Reply:
Here you go:

```json
"HTTP": {
   "inputs": {
      "headers": {
         "Retry-After": "300"
      },
      "method": "GET",
      "uri": https://my-target-url.com
   },
   "runAfter": {},
   "type": "Http"
}
```

Reply:
I still see the same example, which doesn't provide enough context. Please use the fuller example instead, which shows how the HTTP action uses the header:

```json
"HTTP": { 
    "inputs": {
        "headers": {
            "Retry-After": "300"
        }, 
        "method": "GET", 
        "uri": https://my-target-url.com
    }, 
    "runAfter": {},
    "type": "Http" 
}
```